decidim-participatory_processes/app/commands/decidim/participatory_processes/admin/update_participatory_process.rb

```diff
@@ -24,74 +21,33 @@ def initialize(participatory_process, form)
         #
         # Returns nothing.
         def call
-          return broadcast(:invalid) if form.invalid?
-
-          update_participatory_process
+          return broadcast(:invalid) if invalid?
 
-          if @participatory_process.valid?
-            broadcast(:ok, @participatory_process)
-          else
-            form.errors.add(:hero_image, @participatory_process.errors[:hero_image]) if @participatory_process.errors.include? :hero_image
-            form.errors.add(:banner_image, @participatory_process.errors[:banner_image]) if @participatory_process.errors.include? :banner_image
-            broadcast(:invalid)
+          transaction do
+            run_before_hooks
```

Comment:
I think `run_before_hooks` isn't doing anything. I had to check in `UpdateResource` just in case. Maybe its better to drop it? 
```suggestion
```

Reply:
yes, the method is empty. I just kept it to make sure i follow the call method as identical with the base as possible. The scope here is to make sure the attachments ( previously discussed ) are being processed as it should.